tasks/main.yml

```diff
@@ -50,6 +50,16 @@
     - restart node exporter
   when: not ansible_check_mode
 
+- name: Create texfile collector dir
+  file:
+    path: "{{ node_exporter_textfile_dir }}"
+    state: directory
+    owner: "node-exp"
+    group: "node-exp"
+    recurse: true
+    mode: 0755
+  when: node_exporter_textfile_dir is defined
```

Comment:
It will always be defined since it is set in `defaults/main.yml`. Better to check if it is not empty (`node_exporter_textfile_dir != ""`).

Reply:
Sounds good.